### PR TITLE
[admin], use and test dblclick on row from c2cgeoform

### PR DESCRIPTION
**Pull request by cmangeat**
depends on https://github.com/camptocamp/c2cgeoform/pull/162

### CODE REVIEW

File: admin/Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ cleanall:
 .PHONY: test
 test: build ${BUILD_DIR}/requirements-dev.timestamp
 	#Run your project's tests.
-	${VENV}/bin/pytest --driver Firefox
+	${VENV}/bin/pytest --driver Chrome
```
Comment (arnaud-morvan) [inline admin/Makefile]: could you add a line about installing the new driver in readme

File: admin/acceptance_tests/layers_wms_test.py [hunk 2]
```diff
@@ -373,18 +376,33 @@ def test_submit_new_no_layer_name(self, test_app):
                     for x in resp.html.select("[class~'has-error']")])
 
 
+# following test implies use of chrome selenium web driver instead of gecko (Firefox)
```
Comment (arnaud-morvan) [inline admin/acceptance_tests/layers_wms_test.py]: could you put this line exactly where the problem is

File: admin/acceptance_tests/layers_wms_test.py [hunk 3]
```diff
@@ -380,11 +383,25 @@ class TestLayerWMSSelenium():
...
+                'return (window.jQuery != undefined && jQuery.active == 0)'))
+
+        el = index_page.find_element(
+            By.XPATH,
+            '//td[contains(text(),"{}")]'.format(layer.geo_table),
+            timeout=5)
+        action_chains = ActionChains(selenium)
+        # following event implies use of chrome selenium web driver instead of gecko (Firefox)
+        action_chains.move_to_element(el).double_click().perform()
+        assert selenium.current_url.endswith('/layers_wms/{}'.format(layer.id))
```
Comment (arnaud-morvan) [inline admin/acceptance_tests/layers_wms_test.py]: Sorry to be late, but is it possible to factorize this in IndexPage please.

Comment (cmangeat) [reply admin/acceptance_tests/layers_wms_test.py]: You are not late.

**Pull request merged**